src/datascience-ui/interactive-common/redux/reducers/variables.ts

```diff
@@ -238,8 +238,8 @@ function handleRestarted(arg: VariableReducerArg): IVariableState {
                 sortColumn: arg.prevState.sortColumn,
                 sortAscending: arg.prevState.sortAscending,
                 startIndex: 0,
-                pageSize: arg.prevState.pageSize,
-                refreshCount: 0
+                pageSize: 5,
+                refreshCount: arg.prevState.refreshCount
```

Comment:
This here is the actual fix, when we moved from hosting this in each webview view to hosting a single instance in the variable view we changed how we use refreshCount some. If you switch between documents you can sometime have the same execution count as before, in which case we bump up the refresh count to force a refresh. 

However on restart this could lead to a state where we could have the same execution count and the same refresh count. In this case it would not realized that it needed to fetch new data, and you could get odd places where you would get Loading... and the rows would never load. Just keep refreshCount always counting up even after a restart to prevent this.